.azure/gpu-tests-fabric.yml

```diff
@@ -61,7 +61,7 @@ jobs:
           image: "pytorchlightning/pytorch_lightning:base-cuda-py3.10-torch2.2-cuda12.1.0"
           PACKAGE_NAME: "fabric"
         "Lightning | latest":
-          image: "pytorchlightning/pytorch_lightning:base-cuda-py3.10-torch2.2-cuda12.1.0"
+          image: "pytorchlightning/pytorch_lightning:base-cuda-py3.10-torch2.3-cuda12.1.0"
```

Comment:
2.3 was not yet released, so it is not a latest...

Reply:
I'm just using this branch to prepare our code and let us discover any tests that fail.